configure

```diff
@@ -4,25 +4,7 @@
 # Copyright (c) 2007, 2009, 2012 Ramiro Polla
 # Copyright (c) 2014 Tiancheng "Timothy" Gu
 #
-# Parts copied from FFmpeg's configure. Original license was:
-#
-# Copyright (c) 2000-2002 Fabrice Bellard
-# Copyright (c) 2005-2007 Diego Biurrun
-# Copyright (c) 2005-2007 Mans Rullgard
-#
-# dlfcn-win32 is free software; you can redistribute it and/or
-# modify it under the terms of the GNU Lesser General Public
-# License as published by the Free Software Foundation; either
-# version 2.1 of the License, or (at your option) any later version.
-
-# dlfcn-win32 is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-# Lesser General Public License for more details.
-#
-# You should have received a copy of the GNU Lesser General Public
-# License along with dlfcn-win32; if not, write to the Free Software
-# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+# This build system is based in part on the FFmpeg configure script.
```

Comment:
Could you put there explicit license like in other files? Saying just _FFmpeg configure script_ is ambiguous as there were released lot of ffmpeg versions and in future ffmpeg may change license...

Reply:
I wanted to avoid explicitly citing the license by just saying that libvpx has done it and there have been no issues with that phrasing.
But also I don't want to spend too much time on this issue. I'm just ok with relicensing my part of this project to MIT. If this means too much work looking into the licensing for the configure script, I'd prefer to just remove/rewrite it (but I don't want to do this myself).

Reply:
Ok @ramiropolla, thanks a lot!

Reply:
I rephrase the copyright notice in https://github.com/dlfcn-win32/dlfcn-win32/pull/81/commits/d240aff171be5d304e1f0a0af0cb60f4e2e98ecf . The license of the ffmpeg configure script is not clear, but as most of ffmpeg is licensed under lgpl2.1+ I think it is fair to assume that it is licensed under lgpl 2.1 . However, this is not a problem as the viral license of the configure build system does not propagate to the library.